core/arch/arm/kernel/entry_a64.S

```diff
@@ -143,21 +143,15 @@
 	.endm
 
 FUNC _start , :
-#if defined(CFG_CORE_SEL1_SPMC)
 	/*
-	 * With OP-TEE as SPMC at S-EL1 the SPMD (SPD_spmd) in TF-A passes
-	 * the DTB in x0, pagaeble part in x1 and the rest of the registers
-	 * are unused
+	 * If CFG_CORE_FFA x0 if non-NULL holds the TOS FW config address,
+	 * else x0 if non-NULL holds the pagable part address.
```

Comment:
If CFG_CORE_FFA is defined, x0...

What does TOS mean?

Reply:
I'm updating the comment explaining that it's a TF-A concept.